Can't run Node server with run-server.sh

run-server.sh output:
```
npm ERR! Linux 4.4.0-66-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "server-dev"
npm ERR! node v7.7.3
npm ERR! npm  v4.1.2

npm ERR! missing script: server-dev
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/xsolve-feat/server/npm-debug.log
```

--- a/server/run-server.sh
+++ b/server/run-server.sh
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-npm run server-dev
+npm run serve:dev

